src/components/header/index.tsx

```diff
@@ -1,15 +1,110 @@
 import type {FunctionComponent} from 'react'
 import Image from 'next/image'
+
 import {default as NextLink} from 'next/link'
 
+import {useRouter} from 'next/router'
+
+import { faCaretLeft } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+
 const Header: FunctionComponent = () => {
+
+  var routes = ['datasets','models','users','infrastructure','finances','settings'];
+
+  const router = useRouter();
+
+  const list = []
+
+  function myFunc()
+  {
+    var curr_route = router.pathname;
+    var curr_path = curr_route.substring(1);
+
+    if(curr_path=='dashboard')
+    {
+      routes = ['datasets','models','users','infrastructure','finances','settings'];
+    }
+
+    else if(curr_path=='datasets')
+    {
+      routes = ['requests', 'tensors'];
+    }
+
+    else if(curr_path=='users')
+    {
+      routes = ['groups', 'roles'];
+    }
+
+    else if(curr_path=='infrastructure')
+    {
+      routes = ['nodes', 'workers'];
+    }
+
+    else if(curr_path=='settings')
+    {
+      routes = ['infrastructure', 'associations','monetization','binding'];
+    }
+
+    else
+    {
+      routes = [];
+    }
+
+    routes.forEach((route) => {
+      if(curr_path === 'dashboard')
+      {
+        var route_name = route.charAt(0).toUpperCase() + route.slice(1);
```

Comment:
Use the `text-transform: capitalize` property instead. In tailwind, the class is `capitalize`.